cluster-provision/gocli/cmd/run.go

```diff
@@ -499,8 +519,15 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 		if len(nodeQemuArgs) > 0 {
 			additionalArgs = append(additionalArgs, "--qemu-args", shellescape.Quote(nodeQemuArgs))
 		}
-		if len(kernelArgs) > 0 {
-			additionalArgs = append(additionalArgs, "--additional-kernel-args", shellescape.Quote(kernelArgs))
+
+		var hugepagesAmount string
```

Comment:
Maybe I missed something but what's the point of this variable?
Why not just use `hugepagesSize2M` directly?

Reply:
I wanted to leave the structure there in case it was required to support other sizes... (I initially planned to do so). I can definitely trim the code.